docs/host/server setup/windows-server.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Host a Domain Locally on Windows
+---
+
+A local server lets you use your own computer to host your virtual
+domain on the metaverse.
+
+:small_red_triangle_down: Warning
```

Comment:
https://docusaurus.io/docs/next/markdown-features/admonitions

Use the official ways to do this, it's much simpler and cleaner. Also remove the dashes from behind each line, these are sentences, not list items.